fast/stages/1-resman/organization.tf

```diff
@@ -48,60 +52,51 @@ module "organization" {
     (var.tag_names.context) = {
       description = "Resource management context."
       iam         = try(local.tags.context.iam, {})
-      values = {
-        data = {
-          iam         = try(local.tags.context.values.data.iam, {})
-          description = try(local.tags.context.values.data.description, null)
-        }
-        gke = {
-          iam         = try(local.tags.context.values.gke.iam, {})
-          description = try(local.tags.context.values.gke.description, null)
-        }
-        gcve = {
-          iam         = try(local.tags.context.values.gcve.iam, {})
-          description = try(local.tags.context.values.gcve.description, null)
-        }
-        networking = {
-          iam         = try(local.tags.context.values.networking.iam, {})
-          description = try(local.tags.context.values.networking.description, null)
-        }
-        project-factory = {
-          iam         = try(local.tags.context.values.project-factory.iam, {})
-          description = try(local.tags.context.values.project-factory.description, null)
+      values = merge(
+        try(local.tags["context"]["values"], {}),
+        {
+          for k, v in local.tag_values_stage2 : v => {
+            iam         = try(local.tags.context.values.iam[v], {})
+            description = try(local.tags.context.values.description[v], null)
+          } if var.fast_stage_2[k].enabled
         }
-        sandbox = {
-          iam         = try(local.tags.context.values.sandbox.iam, {})
-          description = try(local.tags.context.values.sandbox.description, null)
-        }
-        security = {
-          iam         = try(local.tags.context.values.security.iam, {})
-          description = try(local.tags.context.values.security.description, null)
-        }
-      }
-    }
+      )
+    },
     (var.tag_names.environment) = {
       description = "Environment definition."
       iam         = try(local.tags.environment.iam, {})
       values = {
-        development = {
+        (var.environment_names["dev"]) = {
```

Comment:
Would it be better to iterate `var.environment_names` here and create values for each one?

Reply:
Ah yes, this was before I added the environments variable, good point.

Reply:
pfff you're right :)